Composer/packages/client/src/utils/getPageName.tsx

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { PageNames } from '@bfc/shared';
+
+const PagesRegex = {
+  [PageNames.Design]: /\/dialogs/i,
+  [PageNames.Home]: /\/home|\//i,
```

Comment:
I think this can just be `/\/home/` - the route that's just `/` redirects there, and the way this is written it'll match any path with a slash in it anywhere.